File: java/src/main/java/gherkin/ast/Scenario.java
package gherkin.ast;

import java.util.List;

public class Scenario extends ScenarioDefinition {
    public Scenario(List<Tag> tags, Location location, String keyword, String name, String description, List<Step> steps) {
        super(tags, location, keyword, name, description, steps);
    }

    @Override
    public void accept(Visitor visitor) {
        visitor.visitScenario(this);
    }
}

File: dotnet/Gherkin/Ast/Scenario.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Gherkin.Ast
{
    public class Scenario : ScenarioDefinition, IHasTags
    {
        public IEnumerable<Tag> Tags { get; private set; }

        public Scenario(Tag[] tags, Location location, string keyword, string name, string description, Step[] steps) 
            : base(location, keyword, name, description, steps)
        {
            Tags = tags;
        }
    }
}